pkg/kube/client.go

```diff
@@ -86,7 +86,7 @@ func (c *Client) IsReachable() error {
 	client, _ := c.Factory.KubernetesClientSet()
 	_, err := client.ServerVersion()
 	if err != nil {
-		return errors.New("Kubernetes cluster unreachable")
+		return errors.New("Kubernetes cluster unreachable: " + err.Error())
```

Comment:
can we change this to

```
return fmt.Errorf("Kubernetes cluster unreachable: %s", err.Error())
```

Reply:
Sure. I will change it.

Reply:
I have changed it and amend the original commit.